core/users.js

```diff
@@ -0,0 +1,46 @@
+// import { CastError } from 'mongoose';
+import jwt from 'jsonwebtoken';
+import encryption from 'utils/encryption';
+import { UserNotFoundError, PasswordNotMatchError } from 'common/error';
+import User from 'db/models/User';
+
+const secretKey = process.env.SECRET_KEY;
+
+/**
+ *
+ * Find an user that matches given userId
+ * @param {String} userId
+ * @returns {Object} response data: User
+ */
+async function findById(userId) {
+  let data = {};
+  const user = await User.findOne(
+    { _id: userId },
+    { _id: 0, password_hashed: 0, password_salt: 0 },
+  );
+
+  if (!user) {
+    const err = new UserNotFoundError();
+    throw err;
+  }
+  data = user;
+  return data;
```

Comment:
Unnecessary variable `data`

Reply:
fixed in https://github.com/EasyEventV2/ee2_api/pull/1/commits/94f76d867f0e4b41358e9c942e9fc3c9053d102b